addons/web/static/src/js/services/crash_manager.js

```diff
@@ -19,6 +19,23 @@ var map_title ={
     access_denied: _lt('Access Denied'),
 };
 
+var CrashManagerDialog = Dialog.extend({
+    xmlDependencies: (Dialog.prototype.xmlDependencies || [])
+    .concat(['/web/static/src/xml/crash_manager.xml']),
+    init: function (parent, options) {
+        this._super(parent);
```

Comment:
You forgot to pass `options` to super, you can use `this._super.apply(this, arguments);` to never forget an argument.